packages/python/test/unit.test.ts

```diff
@@ -35,23 +48,33 @@ it('should ignore minor version in vercel dev', async () => {
 });
 
 it('should select latest version when no Piplock detected', async () => {
```

Comment:
This test name needs to be updated if its no longer selecting a specific version

Reply:
Fixed.